lib/dmv_data_service.rb

```diff
@@ -1,25 +1,33 @@
 require 'faraday'
 require 'json'
 
+# The DmvDataService class retrieves external data from an API.
 class DmvDataService
+  # Loads data from a given URL
+  # @param source: String URL for the data source
+  # @return: Parsed JSON data
   def load_data(source)
     response = Faraday.get(source)
     JSON.parse(response.body, symbolize_names: true)
   end
 
+  # Retrieves Washington EV Registration data
   def wa_ev_registrations
-    @wa_ev_registrations ||= load_data('https://data.wa.gov/resource/rpr4-cgyd.json')
+    load_data('https://data.wa.gov/resource/rpr4-cgyd.json')
```

Comment:
I wouldn't expect you to need to update these instance variables. Did you have an issue with reading them from another class?